templates/firefox-ios.xliff

```diff
@@ -4243,9 +4259,35 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
-  <file original="WidgetKit/Base.lproj/WidgetIntents.intentdefinition" source-language="en" target-language="en" datatype="plaintext">
+  <file original="Shared/Supporting Files/en.lproj/JumpBackIn.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="13.4.1" build-num="13F100"></tool>
+    </header>
+    <body>
+      <trans-unit id="ContextualHints.FirefoxHomepage.JumpBackIn.PersonalizedHome" xml:space="preserve">
+        <source>Meet your personalized homepage. Recent tabs, bookmarks, and search results will appear here.</source>
+        <note>Contextual hints are little popups that appear for the users informing them of new features. This one talks about additions to the Firefox homepage regarding a more personalized experience.</note>
+      </trans-unit>
+      <trans-unit id="ContextualHints.FirefoxHomepage.JumpBackIn.SyncedTab.v106" xml:space="preserve">
+        <source>Your tabs are syncing! Pick up where you left off on your other device.</source>
```

Comment:
Should this be plural (device**s**)?

Reply:
Singular might be wanted by design, since there's only one tab
https://www.figma.com/file/7J6gh5D8DbGTExxskKdn3T/Mobile-Onboarding?node-id=1950%3A63885

Reply:
I will check with product